src/py_avro_schema/_schemas.py

```diff
@@ -901,6 +905,10 @@ def _record_field(self, name: str, py_field: pydantic.fields.FieldInfo) -> Recor
         )
         return field_obj
 
+    def make_default(self, py_default: Any) -> Any:
+        """Return an Avro schema compliant default value for a given Python value"""
+        return {key: _schema_obj(value.__class__).make_default(value) for key, value in py_default}
```

Comment:
@faph can we assume it is a pydantic class here so it is iterable like this?

Reply:
@dada-engineer Correct, and the type hint should reflect this.

Also, return type should be `JSONObj`

Reply:
done